spec/features/viewing_a_manual_spec.rb

```diff
@@ -175,22 +175,23 @@
     expect(page.status_code).to eq(404)
   end
 
-  scenario "visiting a withdrawn section" do
```

Comment:
Are we no longer testing withdrawn documents?

Reply:
I think think the terminology of 'withdrawn' is confused with documents that are actually 'gone' so I updated the wording. I did however forget to remove the [stub method](https://github.com/alphagov/manuals-frontend/pull/1113/files#diff-a8c2a229533b0ffd4bf783bbf1a0ae45d28016930dea8062f410d9d112dbb52bL326-L334) that this test used which as you can see from the format is actually a 'gone' document.